components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/internal/DefaultServiceURLBuilder.java

```diff
@@ -79,15 +80,16 @@ public ServiceURLBuilder addPath(String... paths) {
     public ServiceURL build() throws URLBuilderException {
 
         String protocol = fetchProtocol();
-        String hostName = fetchHostName();
+        String proxyHostName = fetchProxyHostName();
+        String internalHostName = fetchInternalHostName();
         int port = fetchPort();
         String tenantDomain = resolveTenantDomain();
         String proxyContextPath = ServerConfiguration.getInstance().getFirstProperty(PROXY_CONTEXT_PATH);
         String resolvedFragment = buildFragment(fragment, fragmentParams);
         String urlPath = getResolvedUrlPath(tenantDomain);
 
-        return new ServiceURLImpl(protocol, hostName, port, tenantDomain, proxyContextPath, urlPath, parameters,
-                resolvedFragment);
+        return new ServiceURLImpl(protocol, proxyHostName, internalHostName, port, tenantDomain, proxyContextPath, urlPath,
```

Comment:
Exceeds 120 characters

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3111/commits/cb3d27b5f55b63844b8f96b218b5eb5e95124c8e

Reply:
This was already fixed

Reply:
This has already fixed